packages/create-ponder/src/helpers/getPackageManager.ts

```diff
@@ -1,21 +1,26 @@
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-import { detect } from "detect-package-manager";
```

Comment:
This was consistently wrong and slow, the checks we have should cover what we need